### DISCUSSION TITLE
Reverse-Connect VNC Guacamole

### DISCUSSION THREAD

**Discussion by Maynardnaze** [htpcBeginner/docker-traefik#217] [Q&A]
Does anyone know the trick to getting Reverse connect to work? id like to keep the vnc port closed and just use guac.
ive gone through and added the reverse connection through mysql

-- Create connection
INSERT INTO guacamole_connection (connection_name, protocol) VALUES ('reverse', 'vnc');
SET @id = LAST_INSERT_ID();

-- Add parameters
INSERT INTO guacamole_connection_parameter VALUES (@id, 'hostname', 'localhost');
INSERT INTO guacamole_connection_parameter VALUES (@id, 'port', '9999');
INSERT INTO guacamole_connection_parameter VALUES (@id, 'reverse-connect', 'true');

Open the connection within Guacamole, then connect to the port on the Guacamole Server with the VNC client (eg. :9999 as shown in the example above). If you don't open the connection within Guacamole first, guacd won't be listening on the given port.

anyone know the correct way to do this?

**Comment by robflate** [comment]
Did you ever get this working?

**Comment by Maynardnaze** [comment]
No, unfortunately. I think using an rdp container might be a better route
then reverse vnc
